corebehrt/common/utils.py

```diff
@@ -74,23 +21,29 @@ class Data:
     censor_outcomes: Optional[List] = field(default=None)
     vocabulary: Optional[Dict] = field(default=None)
     mode: Optional[str] = field(default=None)
-    
+
     def __len__(self):
         return len(self.pids)
-    
-    def copy(self) -> 'Data':
+
+    def copy(self) -> "Data":
```

Comment:
I think we should be able to just `deepcopy(self)` and all attributes should automatically be deepcopied as well

Reply:
Good point! We should note that for the df version of the method.